Tests/SwiftPrettyPrintTests/Public/DebugTests.swift

```diff
@@ -160,16 +172,20 @@ class DebugTests: XCTestCase {
         assertEqualLines(result,
                          """
                          [
-                             "dog-1": Dog(id: "pochi",
-                                          name: "ポチ",
-                                          nickname: nil,
-                                          age: 3,
-                                          homepage: https://www.google.com/),
-                             "dog-2": Dog(id: "pochi",
-                                          name: "ポチ",
-                                          nickname: nil,
-                                          age: 3,
-                                          homepage: https://www.google.com/)
+                             "dog-1": Dog(
+                                         id: "pochi",
+                                         name: "ポチ",
+                                         nickname: nil,
+                                         age: 3,
+                                         homepage: https://www.google.com/
+                                      ),
```

Comment:
Hmmm, should we change dictionary format too for consistency of format? (I didn't think about this)

```swift
"dog-1": Dog(
    id: "pochi",
    name: "ポチ",
    nickname: nil,
    age: 3,
    homepage: https://www.google.com/,

    // dictionary in dictionary
    dict: [
        "apple": Apple(
            price: 100,
            sale: true
        ) ,
        "orange": Orange(
            price: 100,
            sale: true
        ) 
    ]
),
```

What do you think?

Reply:
It's depends on `MultilineFormatter.objectString`
I think this consistency is not bad.

Reply:
Are you say in other words that this problem should fixed but not in this PR?

Reply:
oh, it will be fixed this issue.
https://github.com/YusukeHosonuma/SwiftPrettyPrint/issues/95